files/nbde_client-network-flush

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+for iface in $(ip link | sed -rn 's@^[0-9]+:\s+([[:alnum:]]+):.*@\1@p'); do
```

Comment:
@tyll is there a better way to do this?

Reply:
JSON output of iproute2 would be nicer. But on some systems, that's not available (in particular, CentOS7).

From sysfs might be good. Note there are some special files there like `bonding_masters`. Guess you could do something like `(cd /sys/class/net/; for f in */; do echo ${f%/}; done)` to get rid of that.

but I have no strong preference here.

Sergio, did you strip interfaces with a parent, like `vethf6a43f04@if9`, on purpose?

Reply:
> Sergio, did you strip interfaces with a parent, like `vethf6a43f04@if9`, on purpose?

It was probably accidental. I initially was using `nmcli -t device`, which was simpler to use the output with `cut`, but I think I had issues with CentOS 7, then I switched to this other approach. I like your suggestion (and tyll's) of using `/sys/class/net/`, so I will change to use it instead. Thanks!